src/codegen/utilities/interpreter_helpers.py

```diff
@@ -236,17 +241,20 @@ def get_grpc_interpreter_call_params(func, params):
     has_read_array_parameter = False
     for param in params:
         if param.parameter_name not in compound_params:
-            if is_read_function and "read_array" in param.parameter_name:
+            name = param.parameter_name
+            if param.parameter_name in MODIFIED_INTERPRETER_PARAMS:
+                name = MODIFIED_INTERPRETER_PARAMS.get(param.parameter_name)
+            if is_read_function and "read_array" in name:
                 if has_read_array_parameter:
                     continue
                 grpc_params.append(
                     f"{camel_to_snake_case(param.size.value)}={param.parameter_name}.size"
                 )
                 has_read_array_parameter = True
             elif param.is_grpc_enum or (param.is_enum and not param.is_list):
-                grpc_params.append(f"{param.parameter_name}_raw={param.parameter_name}")
+                grpc_params.append(f"{name}_raw={param.parameter_name}")
             else:
-                grpc_params.append(f"{param.parameter_name}={param.parameter_name}")
+                grpc_params.append(f"{name}={param.parameter_name}")
```

Comment:
Did you intend to replace the other references to `param.parameter_name` with `name`?

Reply:
No Brad, the expectation is to assign `r0 = r_0`, where `param.parameter_name` is `r_0` and the modified name is `r0`